src/components/atoms/OcContextualHelper/OcContextualHelper.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="oc-contextual-helper">
+    <oc-button :id="`${buttonId}`" appearance="raw">
+      <oc-icon name="question" fill-type="line" size="small" />
+    </oc-button>
+    <oc-drop :drop-id="dropId" :toggle="`#${buttonId}`" mode="click" close-on-click>
```

Comment:
No need to do string interpolation here since the computed prop already returns a string (?) ;)

Reply:
in this case it is because of the `#`

Reply:
But then why not turn around the computed prop assignment (generate button ID first and then use it with the prefixed `#` for the drop)? Seems cleaner than doing string interpolation within the template

Reply:
I dont get it